test/helper/utils.js

```diff
@@ -0,0 +1,10 @@
+const chai = require('chai');
+const app = require('../../index')
+
+module.exports.getCsrfTokenHtml = function getCsrfToken(request) {
```

Comment:
```suggestion
module.exports.getCsrfTokenFromPageBody = function getCsrfTokenFromPageBody(request) {
```
(attention il faut renommer la fonction ailleurs aussi)